src/Translation_unit.ml

```diff
@@ -302,12 +307,33 @@ let parse_print (XUnit xunit) (conf : Conf.t) ~input_name ~input_file ic
           let s =
             match m with
             | `Ast -> "ast changed"
-            | `Doc_comment -> "doc comments changed"
+            | `Doc_comment _ -> "doc comments changed"
             | `Comment -> "comments changed"
             | `Comment_dropped _ -> "comments dropped"
           in
           Format.eprintf "  BUG: %s.\n%!" s ;
           ( match m with
+          | `Doc_comment l when not conf.Conf.quiet ->
+              List.iter l ~f:(fun (loc1, loc2, msg) ->
+                  if Location.compare loc1 Location.none = 0 then
+                    Caml.Format.eprintf
```

Comment:
```suggestion
                    Format.eprintf
```

Reply:
I used this one because I want to use `Location.print_loc` printer (the produced output is prettier than with `Location.fmt`)